README.md

```diff
@@ -20,59 +20,60 @@
   </a>
 </p>
 
+*Spring Cloud Data Flow* is a microservices-based streaming and batch data processing pipelines in Cloud Foundry and
+Kubernetes.
 
-*Spring Cloud Data Flow* is a toolkit for building data integration and real-time data processing pipelines. 
-
-Pipelines consist of Spring Boot apps, built using the [Spring Cloud Stream](https://github.com/spring-cloud/spring-cloud-stream)
+Data processing pipelines consist of Spring Boot apps, built using the [Spring Cloud Stream](https://github.com/spring-cloud/spring-cloud-stream)
 or [Spring Cloud Task](https://github.com/spring-cloud/spring-cloud-task) microservice frameworks. 
 
-This makes Spring Cloud Data Flow suitable for a range of data processing use cases, from import/export to 
-event streaming and predictive analytics.
+This makes Spring Cloud Data Flow suitable for a range of data processing use cases, from import/export to event streaming
```

Comment:
Instead of import/export or along with import/export, can we say ETL(Extract, Transform, Load) as this term resonates well with many use cases around task

Reply:
This makes Spring Cloud Data Flow ideal

Reply:
Maybe, but it is still appealing ;)